### DISCUSSION TITLE
Feature request: mlua luau version

### DISCUSSION THREAD

**Discussion by jqnatividad** [mlua-rs/mlua#228] [Ideas]
Currently, luau _VERSION only returns "luau", unlike lua, which also return the version number, e.g.

Luau:
```
> print(_VERSION)
Luau
```

Lua 5.4
```
> print(_VERSION)
Lua 5.4
```

and this was an intentional decision by the Luau team per https://github.com/Roblox/luau/issues/175

Still, it would be nice if mlua luau users can get this information, independent of the luau core team's decision.

Perhaps by exposing the [luau-src cargo.toml version_number](https://github.com/khvzak/luau-src-rs/blob/main/Cargo.toml) in the mlua luau API?

**Comment by khvzak** [comment]
> Still, it would be nice if mlua luau users can get this information, independent of the luau core team's decision.

I agree :) I'll see how I can add/expose it.
